src/genesis.lisp

```diff
@@ -0,0 +1,19 @@
+(in-package :emotiq/config)
+
+(defun genesis/create (configuration &key
+                                       root
+                                       (force nil))
+  (let ((genesis-block-path (merge-pathnames
+                             (alexandria:assoc-value  configuration
+                                                      :genesis-block)
+                             root)))
+    (unless (or force
+                (probe-file genesis-block-path))
+      (let ((genesis-block
+             (cosi/proofs:create-genesis-block
```

Comment:
This is using an out-of-date call signature. This function takes a second arg now. This branch needs rebasing with dev.

Reply:
Addressed via merging with `dev`.